spec/features/add_a_new_peep_spec.rb

```diff
@@ -0,0 +1,9 @@
+feature 'Add' do
```

Comment:
its great that you've separated the features tests across multiple files and for each feature. It makes it so much easier to read/see what you have done especially as this challenge has many components.